src/components/SignUp/Inputs/InputName.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+
+const InputName = (props) => {

```

Comment:
Any differences in behavior between inputName and inputEmail? If not, I'd rather reuse common functionality